<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel</class>
 <widget class="QWidget" name="panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tab_area">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="nucleic_acid_tab">
      <attribute name="title">
       <string>Nucleic Acid</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="domains_tab">
      <attribute name="title">
       <string>Domains</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="update_graphs">
     <property name="toolTip">
      <string>Update graphs</string>
     </property>
     <property name="statusTip">
      <string>Recompute all plots.</string>
     </property>
     <property name="text">
      <string>Update Graphs</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+U</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
